dipy/reconst/dti.py

```diff
@@ -1132,6 +1132,9 @@ def predict(self, gtab, S0=1, step=None):
            The mean non-diffusion weighted signal in each voxel. Default: 1 in
            all voxels.
 
+        step : int
+            The chunk size as a number of voxels.
```

Comment:
We might want to add a bit more explanation here (after the first sentence). Something like: "to increase speed of processing, tensor fitting is done simultaneously over many voxels. This parameter sets the number of voxels that will be fit at once in each iteration. Note that a larger number here should speed things up, but should also take up more memory, so keep an eye on your computers memory consumption as you increase this number.

Also, we should mention that this is an optional parameter, and that it has a default value (10,000).

We should also add all this to the documentation of the `TensorModel` object.